src/main.js

```diff
@@ -175,3 +189,12 @@ function takeBackToMainPosterSection() {
   posterFormSection.classList.add('hidden');
   savedPostersSection.classList.add('hidden');
 }
+
+function showMyCustomPoster() {
+  currentPoster = new Poster(posterImageURLInput.value, posterTitleInput.value, posterQuoteInput.value);
```

Comment:
I'm not sure if this is just the browser, but maybe we can re-format this line so that it's a bit cleaner and easier to read?